src/commands/utility/userinfo.js

```diff
@@ -44,7 +44,7 @@ module.exports = class extends Command {
             .addField('Status', user.presence.status, true)
             .addField('Joined', moment(member.joinedAt).format('MMM DD, YYYY hh:mm A'), true)
             .addField('Registered', moment(user.createdAt).format('MMM DD, YYYY hh:mm A'), true)
-            .addField(`Roles (${member.roles.size - 1})`, `${member.roles.size > 1 ? member.roles.filter(r => r.position !== 0).sort((a, b) => b.position - a.position).map(r => `<@&${r.id}>`).join(', ') : "'None"}`, false)
+            .addField(`Roles (${member.roles.size - 1})`, `${member.roles.size > 1 ? member.roles.filter(r => r.position !== 0).sort((a, b) => b.position - a.position).map(r => `<@&${r.id}>`).join(', ') : 'None'}`, false)
```

Comment:
```suggestion
            .addField(`Roles (${member.roles.size - 1})`, `${member.roles.size > 1 ? member.roles.filter(r => r.position !== 0).sort((a, b) => b.position - a.position).map(r => r.toString()).join(', ') : 'None'}`, false)
```

Reply:
I would also recommend considering removing the entire .filter(). That is an extra loop that could possibly be removed if i understood it right.

If the filter is trying to remove the @ everyone role and its position is 0, would the .sort not put that role all the way at the end of the array every time? So what if it was like

`member.roles.sort().slice(0, -1)`

Just a thought